Ringhel.Procesio.Action.Core/Models/Credentials/FTP/IFtpSftpClient.cs

```diff
@@ -6,5 +6,6 @@ namespace Ringhel.Procesio.Action.Core.Models.Credentials.FTP
     public interface IFtpSftpClient
     {
         Task<List<FTPFilesInformations>> ListFileNamesWithinFolder(string directoryPath, bool isRecursive, string type);
+        Task<bool> RenameFileWithinFolder(string directoryPath, string oldName, string newName);
```

Comment:
rename method to RenameFile
rename directoryPath to folderPath
rename oldName to currentName
add member information